### PR TITLE
feat(layout-grid): Add demo

### LINKED ISSUE #69: Create layout grid component

### PR DESCRIPTION
**Pull request by patrickrodee**
Closes #69

### CODE REVIEW

File: src/ComponentPage.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ import IconToggleCatalog from './IconToggleCatalog';
 import ImageListCatalog from './ImageListCatalog';
 import MenuCatalog from './MenuCatalog';
 import LinearProgressIndicatorCatalog from './LinearProgressIndicatorCatalog';
+import LayoutGridCatalog from './LayoutGridCatalog';
```
Comment (rlfriedman) [inline src/ComponentPage.js]: nit: Move about LinearProgress for abc order

File: src/LayoutGridCatalog.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import ComponentCatalogPanel from './ComponentCatalogPanel.js';
+
+import './styles/LayoutGridCatalog.scss';
+
+const LayoutGridCatalog = () => {
+  return (
+    <ComponentCatalogPanel
+      hero={<LayoutGridHero />}
+      title='Layout Grid'
+      description='Menus display a list of choices on a transient sheet of material.'
```
Comment (rlfriedman) [inline src/LayoutGridCatalog.js]: Update description to be about layout grid

File: src/LayoutGridCatalog.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      <LayoutGrid className='demo-grid demo-grid--alignment' align='right'>
+        <LayoutGridInner>
+          <LayoutGridCell className='demo-cell'></LayoutGridCell>
+          <LayoutGridCell className='demo-cell'></LayoutGridCell>
+          <LayoutGridCell className='demo-cell'></LayoutGridCell>
+        </LayoutGridInner>
+      </LayoutGrid>
+
+      <h3 className='mdc-typography--subheading2'>Cell Alignment</h3>
+      <p>Cell alignment requires a cell height smaller than the grid's inner height.</p>
```
Comment (rlfriedman) [inline src/LayoutGridCatalog.js]: Looks like the ' in grid's needs to be escaped

File: src/LayoutGridCatalog.js (hunk 2 above)
Comment (patrickrodee) [reply src/LayoutGridCatalog.js]: Woops....

**Pull request merged**